streams/src/main/java/org/apache/kafka/streams/kstream/internals/StreamsTopologyGraph.java

```diff
@@ -52,22 +52,22 @@ void writeToTopology(InternalTopologyBuilder topologyBuilder) {
     }
 
 
-    public void addNode(final StreamsGraphNode node) {
+    void addNode(final StreamsGraphNode node) {
```

Comment:
Most of this logic in `addNode` between lines 56 - 70 will either be removed or undergo heavy refactoring.